master/internal/config/config.go

```diff
@@ -146,6 +146,7 @@ type Config struct {
 	Cache                 CacheConfig                       `json:"cache"`
 	Webhooks              WebhooksConfig                    `json:"webhooks"`
 	FeatureSwitches       []string                          `json:"feature_switches"`
+	ReservedPorts         []int                             `json:"reserved_ports"`
```

Comment:
Or DisallowedPorts?

Reply:
Depends upon the perspective.   Either sounds correct to me just depends upon who is reserving or disallowing.
The customer defines them as reserved so that Determined doesn't use them.

> kept specially for a particular purpose or person.
"a reserved seat"

Or the customer tells Determined that they are disallowed for it use.

>  : to deny the force, truth, or validity of 
: to refuse to allow